### PR TITLE
README: Suggest neomake instead of this plugin

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 coala-vim: coala extension for vim/gvim
 =======================================
 
+### Note: We are moving away from this plugin. [neomake](https://github.com/neomake/neomake) is a better alternative with more features.
```
Comment (sils) [inline README.md:4]: that doesn't exist yet. This phrasing sounds like I should look somewhere else

Comment (adtac) [reply README.md:4]: Actually, you just need to add one line to your vimrc when you have neomake installed to make coala work. So actually not much work needed. I'll figure out what that line is a little later :)

Comment (adtac) [reply README.md:4]: Like for example

```
let g:neomake_javascript_jshint_maker = {
    \ 'args': ['--verbose'],
    \ 'errorformat': '%A%f: line %l\, col %v\, %m \(%t%*\d\)',
    \ }
let g:neomake_javascript_enabled_makers = ['jshint']
```

will just get jshint working

**Comment by sils** [comment]
then add the instruction here :)

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

On 10 October 2016 at 16:05, Adhityaa Chandrasekar <notifications@github.com

**Comment by sils** [comment]
@adtac this is really a quick thing, can you update it?

**Comment by adtac** [comment]
Yikes @gitmate-bot slow down :D

Sorry @sils , I somehow didn't get a notif for yout comment. I just got back from my last exam and I'll finish this soon (first I'm doing quickstart however).

**Pull request closed without merging**